src/components/CertificationMetadataForm/index.tsx

```diff
@@ -68,11 +68,7 @@ const CertificationMetadataForm = (props: Props) => {
         ...(!props.isReviewCertification ? {certificationLevel: parseInt(certificationLevel)} : {}),
         ...(!props.isReviewCertification ? {report: report.map(report => report.value)} : {}),
         ...(!props.isReviewCertification ? {subject: subject} : {})
-      },
-      profile: props.isReviewCertification ? { 
-        ...profile, 
-        dapp: subject ? { ...profile?.dapp, subject } : (profile ? profile.dapp : null)
-      } as any : null
```

Comment:
Why is this removed? 

I have some doubts over the master-pull made in this branch - 

1. Make sure PR #30 changes/fixes are retained. I can see https://github.com/input-output-hk/dapps-certification-web/commit/26c454fdd00cfe64b834ec501d2c97262c5064bb#diff-0ac7528762c66094096f63915c8293d4710767d1dedeebf500f086169deb5a06R72-R75 being added there.

Reply:
Ok, I'm working on this

Reply:
Ok, now is added again